10-2.4/Dockerfile

```diff
@@ -15,3 +15,6 @@ RUN mkdir -p /docker-entrypoint-initdb.d
 COPY ./initdb-postgis.sh /docker-entrypoint-initdb.d/postgis.sh
 COPY ./update-postgis.sh /usr/local/bin
 
+COPY ./healthcheck.sh /usr/local/bin/docker-healthcheck
+RUN chmod +x /usr/local/bin/docker-healthcheck
```

Comment:
If you `chmod` the actual file in the repository, this line should not be necessary.